### ISSUE TITLE
♻️ The Module `@truffle/dashboard-hardhat-plugin` Heavily Impacts the `pnpm hardhat` Loading Time

### ISSUE THREAD

**Issue by ChristopherDedominici** [pcaversaccio/hardhat-project-template-ts#103]
### Describe the desired feature:

I noticed that the loading time when running "pnpm hardhat" is quite high, so I investigated a bit.

To measure the loading time I used [hyperfine](https://github.com/sharkdp/hyperfine).
I used this command:
```hyperfine --warmup 3 --runs 10 "pnpm hardhat"```

First, I removed everything from the  'hardhat.config.ts' file. The file is empty. This is the loading baseline time, which is:
<img width="454" alt="image" src="https://github.com/pcaversaccio/hardhat-project-template-ts/assets/18092467/c7f3c51f-9aba-432e-b859-80d53d3ee522">

Then, I kept all the "imports" but I removed all the code logic to measure the modules' loading time. 
Like this:
<img width="541" alt="image" src="https://github.com/pcaversaccio/hardhat-project-template-ts/assets/18092467/b88ddd8c-062c-4ecc-87cc-62e26b173612">
In this scenario, the loading time is:
<img width="463" alt="image" src="https://github.com/pcaversaccio/hardhat-project-template-ts/assets/18092467/d62e24e2-8a61-4c88-bb19-2a1271556c68">

Finally, I just measured the loading time of the package "@truffle/dashboard-hardhat-plugin".
<img width="470" alt="image" src="https://github.com/pcaversaccio/hardhat-project-template-ts/assets/18092467/7943dcda-6036-445f-b6ed-3a618fade8ed">

This package loading time, alone, takes roughly 450-500ms above the baseline mean, which is 660ms (see first screenshot). 
Which is a lot considering that all the modules together take 900-950ms above the baseline mean.
Basically around 50% of the time is consumed by the module "@truffle/dashboard-hardhat-plugin".
Unfortunately, I cannot submit a PR to use slow-imports in the Truffle repo because it's deprecated.

So I was wondering, would it be possible to remove that plugin, or to put it under a conditional import, or some other solution?

Thanks!

**Comment by ChristopherDedominici** [comment]
Adding @fvictorio to keep him in the loop

**Comment by ChristopherDedominici** [comment]
@pcaversaccio thanks for taking a look at it so quickly. The fix looks good to me!

**Comment by pcaversaccio** [comment]
> @pcaversaccio thanks for taking a look at it so quickly. The fix looks good to me!

Any time! If in the future there are similar issues, just open another issue, please.
